src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt

```diff
@@ -98,6 +98,10 @@ class PeerGradingService(
     fun findAllDraxo(response: Response): List<DraxoPeerGrading> =
         peerGradingRepository.findAllByResponseAndType(response, PeerGradingType.DRAXO)
 
+    fun findAllDraxo(sequence: Sequence): List<DraxoPeerGrading> =
+        findAll(sequence)
+            .filterIsInstance<DraxoPeerGrading>()
+
```

Comment:
I don't understand the filter

Reply:
With this method, I just want the DRAXO peer grading.

`DraxoPeerGrading` inherit of `PeerGrading` and the `findAll` method return only `PeerGrading` object. So to ensure I only return `DraxoPeerGrading` I use this filter. 
I could use this filter instead : 
```kotlin
.filter { it is DraxoPeerGrading }
```
But this filter return me a List of PeerGrading (`List<PeerGrading>`) wich will force me to cast it.
The filter I use return me a List of DraxoPeerGrading (`List<DraxoPeerGrading>`). It's more efficient and easier to read.